exercises/concept/census/.docs/hints.md

```diff
@@ -0,0 +1,20 @@
+# Hints
+
+## 1. Create a new resident
+
+- Revisit the [concept:go/structs]() concept for information on how to create structs.
```

Comment:
@ErikSchierboom Does it make sense to use widgets inside of a line of text? I thought they render as rather big boxes once they are actually implemented. I would have expected this to be a normal relative link to e.g. `/tracks/go/concepts/structs`.

Reply:
I'm happy to implement either way here.

Reply:
I used normal relative links to the relevant concept.